hisi-idt.py

```diff
@@ -117,7 +117,7 @@ def sendframe(self, data, loop):
             try:
                 ack = self.s.read()
                 if len(ack) == 1:
-                    if ack == chr(0xaa):
+                    if ack == b'\xaa':
```

Comment:
Doesn't doing a string compare here make the len(ack) == 1 check pointless?

Reply:
Agree. Will remove this